### PR TITLE
OSDOCS-8105: MG and sosreport updated for OVN-K i/c

### PR DESCRIPTION
**Pull request by JoeAldinger**
Version(s):

Issue:
https://issues.redhat.com/browse/OSDOCS-8105

Link to docs preview:
https://65978--docspreview.netlify.app/openshift-enterprise/latest/support/gathering-cluster-data#gathering-data-network-logs_gathering-cluster-data

QE review:
- [x] QE has approved this change.

Additional information:
@pperiyasamy 
@zhaozhanqi

### CODE REVIEW

File: modules/gathering-data-network-logs.adoc [hunk 1]
```diff
@@ -18,6 +18,10 @@ You can gather network logs on all nodes in a cluster.
 $ oc adm must-gather -- gather_network_logs
 ----
 
+[NOTE]
+====
+The `must-gather` tool collects both `nbdb`and `sbdb` databases from all of the nodes in the cluster by default. Adding the `-- gather_network_logs` includes additional `libovsdb` logs containing OVN-Kubernetes transactions.
```
Comment (pperiyasamy) [inline modules/gathering-data-network-logs.adoc:23]:
```suggestion
The `must-gather` tool collects both OVN's `nbdb` and `sbdb` databases from all of the nodes in the cluster by default. Adding the `-- gather_network_logs` includes additional `libovsdb` logs containing OVN-Kubernetes transactions for OVN `nbdb` database.
```

File: modules/support-generating-a-sosreport-archive.adoc [hunk 2]
```diff
@@ -85,6 +85,13 @@ If an existing `toolbox` pod is already running, the `toolbox` command outputs `
 ----
 <1> `-k` enables you to define `sosreport` plugin parameters outside of the defaults.
 +
+.. Optional: run the following command to include all the information in from a node in your report:
```
Comment (pperiyasamy) [inline modules/support-generating-a-sosreport-archive.adoc:88]:
```suggestion
.. Optional: run the following command to include all the information which includes all rotated/compressed  libovsdb logs from a node in your report:
```

File: modules/support-generating-a-sosreport-archive.adoc [hunk 3]
```diff
@@ -85,6 +85,13 @@ If an existing `toolbox` pod is already running, the `toolbox` command outputs `
...
++
+[source,terminal]
+----
+# sosreport --all -logs -k crio.all=on -k crio.logs=on
```
Comment (pperiyasamy) [inline modules/support-generating-a-sosreport-archive.adoc:92]: This must be just `sos report --all-logs`.

File: modules/support-generating-a-sosreport-archive.adoc [hunk 4]
```diff
@@ -85,6 +85,13 @@ If an existing `toolbox` pod is already running, the `toolbox` command outputs `
 ----
```
Comment (pperiyasamy) [inline modules/support-generating-a-sosreport-archive.adoc:85]: `sosreport` is deprecated already with latest toolbox container. 
so i recommend to replace `sosreport` with `sos report`. we still need to check if those -k options still work with sos report command.

File: modules/support-generating-a-sosreport-archive.adoc (hunk 2 above)
Comment (tssurya) [reply modules/support-generating-a-sosreport-archive.adoc:88]: note that the term "libovsdb" is not user facing and hence should be avoided in user facing docs.

Comment (JoeAldinger) [reply modules/support-generating-a-sosreport-archive.adoc:88]: Does the updated wording work?

File: modules/support-generating-a-sosreport-archive.adoc (hunk 4 above)
Comment (JoeAldinger) [reply modules/support-generating-a-sosreport-archive.adoc:85]: @pperiyasamy are you saying that we should be using `sos report` and not `sosreport` in this document?

Comment (pperiyasamy) [reply modules/support-generating-a-sosreport-archive.adoc:85]: yes, correct. it must be `sos report` and remaining arguments are the same.

File: modules/support-generating-a-sosreport-archive.adoc [hunk 5]
```diff
@@ -81,10 +81,17 @@ If an existing `toolbox` pod is already running, the `toolbox` command outputs `
...
-# sosreport -k crio.all=on -k crio.logs=on <1>
+# sos report -k crio.all=on -k crio.logs=on <1>
 ----
 <1> `-k` enables you to define `sosreport` plugin parameters outside of the defaults.
 +
+.. Optional: run the following command to include information on OVN-Kubernetes networking configurations from a node in your report:
++
+[source,terminal]
+----
+# sos report --all -logs
```
Comment (pperiyasamy) [inline modules/support-generating-a-sosreport-archive.adoc:92]:
```suggestion
# sos report --all-logs
```
no space

**Comment by pperiyasamy** [comment]
/lgtm
Thanks @JoeAldinger.

File: modules/support-generating-a-sosreport-archive.adoc [hunk 6]
```diff
@@ -77,14 +77,21 @@ If an existing `toolbox` pod is already running, the `toolbox` command outputs `
...
+.. Run the `sos report` command and enable the `crio.all` and `crio.logs` CRI-O container engine `sosreport` plugins:
 +
 [source,terminal]
 ----
-# sosreport -k crio.all=on -k crio.logs=on <1>
+# sos report -k crio.all=on -k crio.logs=on <1>
 ----
 <1> `-k` enables you to define `sosreport` plugin parameters outside of the defaults.
 +
+.. Optional: run the following command to include information on OVN-Kubernetes networking configurations from a node in your report:
```
Comment (kalexand-rh) [inline modules/support-generating-a-sosreport-archive.adoc:88]:
```suggestion
.. Optional: To include information on OVN-Kubernetes networking configurations from a node in your report, run the following command:
```

You need the capitalization switch, and I think it's better to put the result of the step first.

**Review by kalexand-rh — COMMENTED** [review]
I have a couple of suggestions for clarity, but this change is looking good!

**Review by tssurya — APPROVED** [review]
/lgtm from technical stand point.

TY @pperiyasamy and @JoeAldinger

**Pull request merged**